pkg/testutil/rand/model.go

```diff
@@ -72,7 +72,7 @@ func modelMessage() *model.Message {
 		PluralID:    gofakeit.SentenceSimple(),
 		Message:     gofakeit.SentenceSimple(),
 		Description: gofakeit.SentenceSimple(),
-		Fuzzy:       gofakeit.Bool(),
+		Status:      model.MessageStatus(gofakeit.IntRange(0, 2)), //nolint:gomnd
```

Comment:
You could create `MessageStatus() model.MessageStatus` in `testutil/rand`, I see that repeats in multiple places
then:
```suggestion
		Status:     Status()
```